client/src/pages/EventPage.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Searchbar from "../components/Searchbar";
+import EventList from "../components/EventList";
+import FilterList from "../components/FilterList";
+import { getEvents } from "../api/getEvents";
+import useAsync from "../hooks/useAsync";
+import Footer from "../components/Footer";
+import styled from "@emotion/styled";
+
+const Container = styled.div`
+  display: flex;
+  flex-flow: column;
+  margin-left: 0.8em;
+`;
+
+function EventPage() {
+  const { data: events } = useAsync(getEvents);
```

Comment:
Why don't you import the ERROR and LOADING params from your hook here?

Reply:
well, you gonna need to start somewhere ;-). will fix this